templates/experiments/components/versions/compare.html

```diff
@@ -1,16 +1,42 @@
 <div class="my-2">
     {% if version_details.fields_changed %}
-        {% for field in version_details.fields %}
-            <div class="grid grid-cols-2 gap-2 max-h-64 overflow-y-auto">
-                {% include 'experiments/components/versions/version_field_details.html' with field=field.previous_field_version field_changed=field.changed diff_color='bg-red-400' %}
-                {% include 'experiments/components/versions/version_field_details.html' with field=field field_changed=field.changed diff_color='bg-emerald-400' %}
-            </div>
+        {% for group_name, group in version_details.fields_grouped %}
+            {% if group.show %}
+                <div x-data="{ isOpen: {{ group.has_changed_fields|yesno:'true,false' }} }" class="mb-2">
```

Comment:
Looking at this some more I don't think alpine is necessary here at all. I think the same can be achieved using the 'checkbox' collapse and setting the checkbox to 'checked' if you want the contents to be initially visible: https://daisyui.com/components/collapse/#collapse-with-checkbox

Reply:
I remember previously I did try this, but my previous approach to achieving default open/close behaviour was wrong, which is why I resorted to using AlpineJS. Using the `checked` property solves this. Thanks for the suggestion

Reply:
Nice, the updated version is much better!